substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/phases/IntrinsifyMethodHandlesInvocationPlugin.java

```diff
@@ -504,6 +512,12 @@ private void processInvokeWithMethodHandle(GraphBuilderContext b, Replacements r
         }
     }
 
+    public List<Pair<ResolvedJavaMethod, Integer>> getContext(GraphBuilderContext b) {
+        List<Pair<ResolvedJavaMethod, Integer>> callingContext = new ArrayList<>();
```

Comment:
This can be a singleton list (`Collecitons.singletonList()`).

Reply:
done